js-api-spec/legacy/importer.node.test.ts

```diff
@@ -935,3 +936,35 @@ describe('when importer returns non-string contents', () => {
     );
   });
 });
+
+// Regression test for sass/dart-sass#1962
+it('compiles multiple nested relative imports loaded multiple times across different files', async () => {
+  await sandbox(dir => {
+    dir.write({
+      'src/index.scss': `
+        @import "./_a.scss";
+        @import "./_includes.scss";
+      `,
+      'include/_a.scss': '/* A */',
+      'include/_b.scss': '/* B */',
+      'src/_includes.scss': `
+        @import "./_a.scss";
+        @import "./_b.scss";
+      `,
+    });
+
+    const result = sass.renderSync({
+      file: dir('src/index.scss'),
+      importer: function (url) {
+        const path = dir('./include', url);
+        if (fs.existsSync(path)) {
+          return {
+            contents: fs.readFileSync(path, 'utf8'),
+          };
+        }
+        return null;
+      },
+    });
+    expect(result.css.toString()).toBe('/* A */\n/* A */ /* B */');
```

Comment:
Note: once https://github.com/sass/dart-sass/pull/1989 lands, this will (correctly) become

```suggestion
    expect(result.css.toString()).toBe('/* A */\n/* A */\n/* B */');
```

Reply:
Even after syncing past that commit I don't see a change in behavior (i.e. whitespace instead of `\n` is still working)

Reply:
nvm I see this problem happening now :)